pmoves/services/hi-rag-gateway-v2/rerank.py

```
@@ -0,0 +1,232 @@
+    """
+    import config as _cfg
+    with _reranker_lock:
+        old_model = _cfg.RERANK_MODEL
+        old_type = _cfg.RERANKER_TYPE
+        # Unload current model to free GPU memory
+        if _cfg._reranker is not None:
+            try:
+                if hasattr(_cfg._reranker, "model") and hasattr(_cfg._reranker.model, "to"):
+                    _cfg._reranker.model.to("meta")
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)